setup.py

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+from setuptools import setup, find_packages
+
+
+setup(
+    name='OpenFisca-Italy',
+    version='2.1.0',
```

Comment:
It seems odd to start a new model at version `2.1.0` :)

`OpenFisca-Core` and `OpenFisca-France` follow [semantic versioning](https://semver.org/).

SemVer is a set of rules about versioning that helps re-users to know whether they can upgrade their dependency without fearing to break something. The two main rules are
- we bump the left-most number for and only for non-backward compatible changes.
- If the left-most number is a 0, then backward compatibility is not guaranteed between two versions.

For instance, there can be a non backward compatible change (breaking change) from `1.1.2` to `2.0`, from `0.1.0` to `0.1.1`, but not from `1.1.2` to `1.2`

At an early stage, you probably don't want to bother with backward compatibility: you don't have users yet. To indicate that your code doesn't guarantee  backward compatibility, I'd suggest starting with `0.1.0`.